procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb

```diff
@@ -5,7 +5,7 @@
    "id": "3dbf3c0c",
```

Comment:
<p>Seeing as these are not be done regularly, and they'll be removed in the future, please leave them commented.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>Moved to the end <code>Quick Troubleshooting</code> section </p>